### DISCUSSION TITLE
Skyline plugin has panicked + Software is aborting

### DISCUSSION THREAD

**Discussion by Nick-Vasquez-CSU** [Raytwo/ARCropolis#346] [Issues] [answered]
System Firmware Version
14.1.1

Atmosphere Version
1.3.1

Smash Version
13.0.1

ARCropolis Version
3.4.0

Can the crash be consistently reproduced?
Yes

At what moment does the crash happen?
Starting up the game

Details:
This problem occurred when I was on arc 3.3.0 as well. I made no changes to any of my mods and was met with this error code: 
![2022051722072800-1AB131B6E6571375B79964211BB3F5AE](https://user-images.githubusercontent.com/42751266/168974366-448b0da6-a213-49a8-9dbd-baede1000ac9.jpg) Subsequently after, I was met with the following crash report either (I'm not sure if they are related or two separate matters):
Crash report
[01652855072_01006a800016e000.log](https://github.com/Raytwo/ARCropolis/files/8713798/01652855072_01006a800016e000.log)
Every time I start up smash, I am met with both the skyline panic and the software closing due to an error. I have already updated skyline.

**Comment by Raytwo** [comment, marked as answer]
Your configuration file seems busted, delete sd:/ultimate/arcropolis/config and boot the game again.

**Reply by Nick-Vasquez-CSU** [reply]
Deleting the config file has cleared my issues. Thank you.
